content/MinimumCD/content-contributors.md

```diff
@@ -29,26 +29,27 @@ Read our [contribution guidelines](https://github.com/Minimum-CD/cd-manifesto/bl
 [Nick&nbsp;Eggleston](https://www.linkedin.com/in/nick-eggleston-light/)&nbsp;-
 [Alessandro&nbsp;Fardin](https://www.linkedin.com/in/alessandro-fardin-61028b28/)&nbsp;-
 [Dave&nbsp;Farley](https://www.linkedin.com/in/dave-farley-a67927)&nbsp;-
-[Javier&nbsp;Lopez&nbsp;Fernandez](javisan81@hotmail.com)&nbsp;-
+[Javier&nbsp;Lopez&nbsp;Fernandez](mailto:javisan81@hotmail.com)&nbsp;-
 [Bryan&nbsp;Finster](https://www.linkedin.com/in/bryan-finster/)&nbsp;-
 [Tiago&nbsp;Gabriel](https://dev.to/tgdraugr)&nbsp;-
 [Chris&nbsp;Gallivan](https://www.linkedin.com/in/christopher-gallivan-16a2b02/)&nbsp;-
 [Chris&nbsp;Gossett](https://www.linkedin.com/in/christopher-gossett-03b09347/)&nbsp;-
 [Nathen&nbsp;Harvey](https://twitter.com/nathenharvey)&nbsp;-
 [Dave&nbsp;Hawes-Johnson](https://www.linkedin.com/in/davidhawesjohnson/)&nbsp;-
-[Adam&nbsp;Hawkins](adam@hawkins.io)&nbsp;-
+[Adam&nbsp;Hawkins](mailto:adam@hawkins.io)&nbsp;-
 [Ferrix&nbsp;Hovi](https://www.linkedin.com/in/ferrix/)&nbsp;-
 [Patrick&nbsp;S.&nbsp;Kelso](<https://www.linkedin.com/in/patrickkelso/>)&nbsp;-
-[Jan&nbsp;Krag](jankrag@gmail.com)&nbsp;-
+[Jan&nbsp;Krag](mailto:jankrag@gmail.com)&nbsp;-
 [Andrea&nbsp;Laforgia](https://www.linkedin.com/in/andrealaforgia/)&nbsp;-
-[Jean-François&nbsp;Lépine](lepinejeanfrancois@gmail.com)&nbsp;-
+[Jean-François&nbsp;Lépine](mailto:lepinejeanfrancois@gmail.com)&nbsp;-
 [Jesse&nbsp;Lin](https://www.linkedin.com/in/jesse-lin/)&nbsp;-
 [Javier&nbsp;A&nbsp;Magaña](https://www.linkedin.com/in/javier-a-magana-98108/)&nbsp;-
 [Jerreck&nbsp;McWilliams](https://www.linkedin.com/in/jerreck/)&nbsp;-
 [Nathan&nbsp;Nicholson](https://www.linkedin.com/in/nathan-r-nicholson/)&nbsp;-
 [Rosalind&nbsp;Radcliffe](https://www.linkedin.com/in/rosalind-radcliffe/)&nbsp;-
 [Christina&nbsp;Rhylander](https://www.linkedin.com/in/christina-rhylander-78683495/)&nbsp;-
 [Prasanjit&nbsp;Singh](https://www.linkedin.com/in/prasanjit-singh/)&nbsp;-
-[Emiliano&nbsp;Sutil](https://www.linkedin.com/in/emiliano-sutil-77a2091b/)
+[Emiliano&nbsp;Sutil](https://www.linkedin.com/in/emiliano-sutil-77a2091b/)&nbsp;-
+[Austin&nbsp;Abro](https://www.linkedin.com/in/austinabro/)
```

Comment:
The list is alphabetized by last name. ;)

Reply:
Okay should be good now. Side note can't believe there are two people ahead of me. I instinctively put myself at the top then realized lol